BlueDBOnDisk/src/main/java/org/bluedb/disk/recovery/RecoveryManager.java

```diff
@@ -108,6 +110,10 @@ public void markChangePending(Path completedPath) throws BlueDbException {
 		return FileUtils.getFolderContentsExcludingTempFiles(historyFolderPath, SUFFIX_COMPLETE);

```

Comment:
I don't want this method anymore. I don't think we should have any methods for getting pending or completed change files that will have problems if there are too many files in the directory if we can help it.

Reply:
Handled in next commit